schema/src/BT01ProjectInfo.yaml

```diff
@@ -57,69 +78,67 @@ classes:
-    range: float
-    description: The ecological connectivity index of the project.
-  socialCulturalIndex:
-    range: float
-    description: The social cultural index of the project.
-  managementAreas:
-    range: ManagementArea
-    description: The management areas associated with the project.
-    multivalued: true
-    inlined: true
-  projectOperator:
-    range: ProjectRole
-    description: The organization responsible for operating the project.
-  projectOwner:
-    range: ProjectRole
-    description: The organization responsible for owning the project.
-  projectDuration:
-    range: string
-    description: The duration of the project.
-  conservationStatus:
-    range: ConservationStatusTypes
-    description: The conservation status of the project.
-
 enums:
-  MarketTypeTypes:
-    description: The type of marketplace
+  BT01RegionTypes:
+    description: The regions for BT01 projects.
     permissible_values:
-      COMPLIANCE_MARKET:
-        meaning: rfs:ComplianceMarket
-        description: a compliance market
-      VOLUNTARY_MARKET:
-        meaning: rfs:VoluntaryMarket
-        description: a voluntary market
-  # TODO: these should be imported from a shared vocabulary
+      ANDEAN:
+        description: Andean
+      CARIBBEAN:
+        description: Caribbean
+      ORINOCO:
+        description: Orinoco
+      PACIFIC:
+        description: Pacific
+  BT01BioregionTypes:
+    description: The bioregions for BT01 projects.
+    permissible_values:
+      LOW_MONTANE_VERY_HUMID_FOREST:
+        description: Low Montane Very Humid Forest
+      ORINOCO_SAVANNA:
+        description: Orinoco Savanna
+      PREMONTANE_HUMID_FOREST:
+        description: Premontane Humid Forest
+      RIPARIAN_FOREST:
+        description: Riparian Forest
+      SAVANNA:
+        description: Savanna
+      TROPICAL_HUMID_FOREST:
+        description: Tropical Humid Forest
+      TROPICAL_DRY_FOREST:
+        description: Tropical Dry Forest
+  BT01BiomeTypes:
+    description: The biome types for BT01 projects.
+    permissible_values:
+      ANDEAN_OROBIOME_NORTH_PACIFIC_FOOTHILLS:
+        description: Andean Orobiome North Pacific Foothills
+      LARGE_TOLIMA_HYDROBIOME:
+        description: Large Tolima hydrobiome
+      LARGE_TOLIMA_SUBANDEAN_OROBIOME:
+        description: Large Tolima Subandean Orobiome
+      TROPICAL_HUMID_ZONOBIOME_TOLIMA_GRANDE:
+        description: Tropical Humid Zonobiome Tolima Grande
+  BT01WatershedTypes:
+    description: The watersheds for BT01 projects.
+    permissible_values:
+      FRIO_RIVER:
+        description: Frío river
+      SUMAPAZ_RIVER_LOWER_WATERSHED:
+        description: Sumapaz river - lower watershed
+  BT01SubWatershedTypes:
+    description: The watersheds for BT01 projects.
+    permissible_values:
+      RIO_FRIO_Y_OTROS_DIRECTOS_AL_CAUCA:
+        description: Río Frío y otros Directos al Cauca
+      RIO_SUMAPAZ:
+        description: Río Sumapaz
```

Comment:
There are some inconsistencies in watershed/subwatershed enum values. I also wonder if we could just use a single enum for both of these. Ideally we could do some kind of hierarchy but prob not necessary for now.

- Usage of `rio` vs `river`
- I think `Sumpaz river - lowerwater shed` is the _sub-watershed_, and `Rio Sumapaz` is the watershed? (these are the values I saw for [BT01-002 metadata](https://regenscan.netlify.app/project/BT01-002))